sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/CombineFns.java

```diff
@@ -996,7 +996,11 @@ public void encode(Object[] value, OutputStream outStream, Context context)
       checkArgument(value.length == codersCount);
       Context nestedContext = context.nested();
       for (int i = 0; i < codersCount; ++i) {
```

Comment:
Ditto on using iterator to differentiate the loop from the last element case as done in MapCoder